### PR TITLE
1.98.0 rc.4.2

### PR DESCRIPTION
**Pull request by linhlam-kc**
PR to update to 1.98.0

### CODE REVIEW

File: README-install-guide.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ helm upgrade --install kubecost \
  --namespace kubecost --create-namespace \
  -f https://raw.githubusercontent.com/kubecost/openshift-helm-chart/1.98.0-rc.4.1/cost-analyzer/disable-psps.yaml \
  -f https://raw.githubusercontent.com/kubecost/openshift-helm-chart/1.98.0-rc.4.1/cost-analyzer/values-thanos.yaml \
```
Comment (linhlam-kc) [inline README-install-guide.md]: @jessegoodier For single cluster deployment, do we need to enable Thanos?

File: README-install-guide.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ helm upgrade --install kubecost \
  --namespace kubecost --create-namespace \
  -f https://raw.githubusercontent.com/kubecost/openshift-helm-chart/1.98.0-rc.4.1/cost-analyzer/disable-psps.yaml \
```
Comment (linhlam-kc) [inline README-install-guide.md]: @jessegoodier I think we should combine disable-psps.yaml, values-restricted-podsecurity.yaml and kubecost-primary-cluster-settings.yaml into 1 values file such as values-openshift.yaml to simplify the installation command and align with what we have in previous version.

**Review by linhlam-kc — COMMENTED** [review]
For summary, these are my comments/questions:

1. I think we should combine disable-psps.yaml, values-restricted-podsecurity.yaml and kubecost-primary-cluster-settings.yaml into 1 values file such as values-openshift.yaml to simplify the installation command and align with what we have in previous version.

2. For single cluster deployment, do we need to enable Thanos?

**Comment by linhlam-kc** [comment]
@jessegoodier In my [latest commit](https://github.com/kubecost/openshift-helm-chart/pull/9/commits/0b0d420c648d5443d4ebbc535db0eba019f52e54) I have fixed prometheus/templates/server-deployment.yaml and add a values-openshift.yaml to simplify the single cluster deployment. I have tested the deployment on OCP 4.11.20. Please check and let me know if it looks good to you before we merge it into the main branch.

**Comment by jessegoodier** [comment]
Nice work. Will test tomorrow morning.

File: cost-analyzer/values-unrestricted-podsecurity.yaml [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
-      - ALL
-    runAsUser: 1001070000
-    runAsNonRoot: true
-    allowPrivilegeEscalation: false
+    # capabilities:
+    #   drop:
+    #   - ALL
+    fsGroup: 1000980000
+    runAsGroup: 1000980000
+    runAsUser: 1000980000
```
Comment (linhlam-kc) [inline cost-analyzer/values-unrestricted-podsecurity.yaml:8]: @jessegoodier Fyi, in my testing, it is good to use runAsNonRoot: true here without setting up static value for `runAsUser`, `runAsGroup`, `fsGroup` in 4.11

**Pull request merged**